base_user_role/wizard/create_from_user.py

```diff
@@ -0,0 +1,46 @@
+from odoo import models, fields
+
+
+class CreateFromUserWizard(models.TransientModel):
+    _name = "base_user_role.create_from_user_wizard"
```

Comment:
```suggestion
class WizardCreateRoleFromUser(models.TransientModel):
    _name = "wizard.create.role.from.user"
```

proposing name more simple.